test/gscraper/accounts/user_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule Gscraper.Accounts.UserTest do
+  use Gscraper.DataCase
+
+  alias Gscraper.Accounts.User
+
+  describe "changeset" do
```

Comment:
Let's add a function arity, `changeset/2`

Reply:
Fixed in 7c36f2d.